crates/nomination/src/lib.rs

```diff
@@ -266,44 +283,77 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
-    pub fn request_nominator_withdrawal(
-        operator_id: &T::AccountId,
-        nominator_id: &T::AccountId,
-        collateral_to_withdraw: Backing<T>,
-    ) -> DispatchResult {
+    pub fn cancel_operator_withdrawal(operator_id: &T::AccountId, request_id: &H256) -> DispatchResult {
+        ensure!(
+            Self::is_operator(&operator_id)?,
+            Error::<T>::VaultNotOptedInToNomination
+        );
         let mut operator = Self::get_rich_operator_from_id(operator_id)?;
-        let request_id = ext::security::get_secure_id::<T>(operator_id);
-        let height = ext::security::active_block_number::<T>();
-        let maturity = height
-            .checked_add(&Self::get_nominator_unbonding_period())
-            .ok_or(Error::<T>::ArithmeticOverflow)?;
-        operator.add_pending_nominator_withdrawal(
-            nominator_id.clone(),
-            request_id,
-            collateral_to_withdraw,
-            maturity,
-        )?;
-        Self::deposit_event(Event::<T>::RequestNominatorCollateralWithdrawal(
-            request_id,
-            nominator_id.clone(),
+        operator.remove_pending_operator_withdrawal(*request_id)?;
+        Self::deposit_event(Event::<T>::CancelOperatorCollateralWithdrawal(
+            *request_id,
             operator_id.clone(),
-            maturity,
-            collateral_to_withdraw,
         ));
         Ok(())
     }
 
-    pub fn execute_nominator_withdrawal(operator_id: &T::AccountId, nominator_id: &T::AccountId) -> DispatchResult {
-        let mut operator = Self::get_rich_operator_from_id(operator_id)?;
-        let matured_collateral = operator.execute_nominator_withdrawal(nominator_id.clone())?;
-        ensure!(!matured_collateral.is_zero(), Error::<T>::NoMaturedCollateral);
-        Self::deposit_event(Event::<T>::ExecuteNominatorCollateralWithdrawal(
-            nominator_id.clone(),
-            operator_id.clone(),
-            matured_collateral,
-        ));
-        Ok(())
-    }
+    // pub fn request_nominator_withdrawal(
```

Comment:
Maybe we can just remove these functions for now instead of commenting them out?